src/index.js

```diff
@@ -6,3 +6,33 @@ import "./index.scss";
 
 // install snippets styles
 import "highlight.js/styles/default.css";
+
+window.onload = () => {
+  const localeSeletor = document.getElementById("localeSelector");
+  localeSeletor.innerHTML = isZhCN() ? 'English' : "中文";
+  localeSeletor.addEventListener("click", onLocaleChange);
+};
+
+const onLocaleChange = () => {
+  const currentProtocol = `${window.location.protocol}//`;
```

Comment:
Consider using `window.location.origin`, which includes the protocol and hostname together. It also does port, if you're running a local dev env